src/Components/Search/NewSearch/Mobile/SearchResultsList.tsx

```diff
@@ -18,11 +18,16 @@ import {
 import { InfiniteScrollSentinel } from "Components/InfiniteScrollSentinel"
 import { NoResults } from "Components/Search/NewSearch/Mobile/SearchResultsList/NoResults"
 import { ContentPlaceholder } from "Components/Search/NewSearch/Mobile/SearchResultsList/ContentPlaceholder"
+import { useTracking } from "react-tracking"
+import * as DeprecatedSchema from "@artsy/cohesion/dist/DeprecatedSchema"
```

Comment:
Why are we still using the `DeprecatedSchema` here — can't we update cohesion?

Reply:
addressed, thanks 👍